lib/facebook.ex

```diff
@@ -213,6 +213,77 @@ defmodule Facebook do
     Facebook.Graph.get(~s(/#{page_id}/feed), params)
   end
 
+  @doc """
+  Gets the total number of people who liked an object.
+  An *object* stands for: post, comment, link, status update, photo.
+
+  If you want to get the likes of a page, please see *pageLikes*.
+
+  ## Example
+      iex> Facebook.objectCount(:likes, "1326382730725053_1326476257382367", "<Token>")
+      2
+
+  See: https://developers.facebook.com/docs/graph-api/reference/object/likes
+  """
+  def objectCount(:likes, object_id, access_token) do
+    :likes
+      |> Atom.to_string
+      |> objectSummary(object_id, access_token)
```

Comment:
actually, now that I think about it.. Why do you pass :likes as an atom when you convert it to string anyway? 
Couldn't you just do:
```
objectSummary("likes", object_id, access_token)
  |> summaryCount
```